airbyte-integrations/connectors/source-the-guardian-api/unit_tests/test_paginator.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+import logging
+from unittest.mock import MagicMock
+
+import airbyte_cdk
+import pytest
+import requests
+from airbyte_cdk.test.utils.manifest_only_fixtures import components_module, connector_dir
```

Comment:
❓ Why do we have to import?

Should this not be a pytest plugin that adds the fixture as an option for any test?

e.g. 
```py
# conftest.py
pytest_plugins = ["airbyte_cdk.test_fixtures"]
```

```py
# some_unit_test.py
def test_ connector_dir(connector_dir):
    assert connector_dir.exists()
```

Reply:
Yup, this approach seems to work and allows us to remove the import 👍